components/select/demo/api.md

```diff
@@ -17,7 +17,7 @@ The auro-select element is a wrapper for auro-dropdown and auro-menu to create a
 | Property                        | Attribute                       | Type      | Default                | Description                                      |
 |---------------------------------|---------------------------------|-----------|------------------------|--------------------------------------------------|
 | [disabled](#disabled)                      | `disabled`                      | `Boolean` |                        | When attribute is present element shows disabled state. |
-| [error](#error)                         | `error`                         | `String`  |                        | When defined, sets persistent validity to `customError` and sets `setCustomValidity` = attribute value. |
+| [error](#error)                         | `error`                         | `String`  |                        | When defined, sets persistent validity to `customError` and sets the validation message to the attribute value. |

```

Comment:
We should have similar text in the doc page example. Similar comment was added elsewhere in this review.

Reply:
Fixed in all components.